test/run-all

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+id=$(id -u)
+
+needs_root() {
+    grep -q NEED_ROOT "$1"
+}
+
+
+tests=( )
+for t in ./test/test-*; do
+    [ -x "$t" ] && [ -f "$t" ] && tests[${#tests[@]}]="$t"
```

Comment:
What's the ```[ -f "$t" ]``` for? doesn't -x cover it? Or do you expect executable non-files in the tree?

Reply:
well... if there were a directory there 'test/test-helpers/' then it would match,
so avoid that.